data_model/src/asset.rs

```diff
@@ -367,10 +402,10 @@ impl AssetDefinition {
 
     /// Token definition with decimal quantity asset value type.
     #[inline]
-    pub fn new_fixed_precision_token(
+    pub fn with_precision_token(
```

Comment:
name of this method goes against the convention that is used for other methods. Can we make this fn or other constructors fall in line? I don't mind either way just so long as it is consistent

Reply:
I'll implement in this PR.